tests/tree_util_tests.py

```diff
@@ -15,11 +15,13 @@
 
 import collections
 
+from unittest import skipIf
 from absl.testing import absltest
 from absl.testing import parameterized
 
 from jax import test_util as jtu
 from jax import tree_util
+from jax.lib import version as jaxlib_version
```

Comment:
Why not just `import jax` and then use `jax.lib.version`? One character longer, and doesn't require defining a new name.